### PR TITLE
Add "all samples destroyed" to project timeline

### PR DESCRIPTION
**Pull request by sofiahag**
As requested by Maria.

### CODE REVIEW

**Comment by sofiahag** [comment]
See e.g. projects P19501 and P1125

**Comment by sofiahag** [comment]
Up on stage!

File: run_dir/static/js/project_samples.js [hunk 1]
```diff
@@ -393,6 +393,10 @@ function load_all_udfs(){
         }
       }
 
+      else if (prettify(key) == 'all_samples_destroyed'){
+       $('#all_samples_destroyed').text(value);
+      }
+
```
Comment (aanil) [inline run_dir/static/js/project_samples.js:399]: Why was this needed?

Comment (sofiahag) [reply run_dir/static/js/project_samples.js:399]: What?? That's weird, I removed the `rawdate` class from the HTML-file, since the date only showed up for aborted projects if I kept it. I thought that was the "fix"-commit 😆 
Very weird...I already had the lines from the JS-file, I never removed them, obviously...

Comment (aanil) [inline run_dir/static/js/project_samples.js:399]:
```suggestion
```

**Review by aanil — APPROVED** [review]
👍
